serverless.yaml

```diff
@@ -106,7 +106,19 @@ functions:
               - 'profile'
               - 'aws.cognito.signin.user.admin'
           method: ANY
-          path: '/{proxy+}'
+          path: '/'
```

Comment:
I'm not sure if this will make a difference, but when testing, I didn't require quotes around `/`. 

It just looked like this.
`path: /`

https://www.serverless.com/framework/docs/providers/aws/events/apigateway/

Reply:
@nguyen102, that's just YAML syntax that allows unquoted strings. The following are equivalent:

yaml
```yaml
string1: /
string2: '/'
string3: "/"

```
json
```json
{
  "string1": "/",
  "string2": "/",
  "string3": "/"
}
```